docs/user-guide/installation/helics_cmake_options.md

```diff
@@ -58,6 +58,7 @@ Options effect the connection of libraries used in HELICS and how they are linke
 - `HELICS_ENABLE_GIT_HOOKS`: install a git hook to check clang format before a push
 - `Boost_NO_BOOST_CMAKE`: \[Default=OFF\] This is an option related to the Boost find module, but is occasionally needed if a specific version of boost is desired and there is a system copy of BoostConfig.cmake. So if an incorrect version of boost is being found even when `BOOST_ROOT` is being specified this option might need to be set to `ON`.
 - `HELICS_BUILD_CONFIGURATION`: A string containing a specialized build configuration if any. The only platform this is currently used on is for building on a Raspberry PI system, in which case this should be set to "PI".
+- `HELICS_DISABLE_C_SHARED_LIB`: \[Default=OFF\] Turns off building of the HELICS C shared library. Probably very few use cases where a user/developer would want to turn this option ON.
```

Comment:
I use this option fairly regularly building the apps that use the C++ shared library, and in the docker containers that generate the apps.  I am fine as is but if you change it, can you remove the HELICS_BUILD_MATLAB_INTERFACE line from this file as well.

Reply:
I reworded it to describe that scenario and mention that use of the C++98 wrapper requires the C shared library.